apps/web-giddh/src/app/tb-pl-bs/components/pl/pl-grid/pl-grid.component.scss

```diff
@@ -0,0 +1,64 @@
+@import "../../../../../assets/css/color.scss";
+   :host ::ng-deep .table-container {
+      padding: 0;
+    }
+
+    :host ::ng-deep .table-container .profitLoss section div > div {
```

Comment:
Avoid camelCase in class names

Reply:
profitLoss class and text is appearing many palces (28 files) including TS file , so I am avoiding this class name